Assets/Scripts/GameElements/PlayerController.cs

```diff
@@ -184,6 +203,27 @@ private void CheckForWingInput()
         }
     }
 
+    private void CheckForPlayerMash()
+    {
+        // Check if any key has been pressed and then any key released
+        // TODO: not great as if, improvement will be to do wing keys
+        // TODO: further improvement would be to detect flaps
+        // TODO: further further improvment is to rely on time - faster flaps = shorter trap time
+        if (buttonPressed != Input.anyKey)
```

Comment:
You may want to do Input.anyKeyDown, since that would bypass this `buttonPressed` variable entirely!